_pages/catering-menus.md

```diff
@@ -7,40 +7,163 @@ description: |
     Soups up!
 ---
 
-## Continental Breakfast - All Days
+## Afternoon Breaks
 
-- Chilled Orange, Apple, and Cranberry Juices
+- Assorted Granola Bars, GF Options available
+- Assorted Freshly Baked Cookies & Brownies, GF Options available
 - Seasonal Fresh Fruit
-- Freshly Baked Breakfast Breads, Pastries, Butter, and Fruit Preserves
-  (non-dairy spread option available; gluten-free option available)
-- Freshly Brewed Coffee, Decaffeinated Coffee, and Selection of Teas
+- Assorted Pepsi Soft Drinks
+- Freshly Brewed Coffee, Decaffeinated Coffee, & Selection of Teas
 
 ----
 
-## Tutorials: Sunday, September 22 - Lunch
+## Breakfast
 
-### TBD
+### Sunday - Wednesday Breakfasts
 
-----
+- Chilled Orange, Apple & Cranberry Juices
```

Comment:
Inconsistent use of oxford comma vs line 16 above and lines 26-27 below

Reply:
updated, thanks